test/release/examples/benchmarks/hpcc/variants/README

```diff
@@ -11,6 +11,7 @@ annual HPCC competition).
 The main files for these benchmarks are as follows:
 
      stream-promoted.chpl   : global STREAM Triad using promotion
+     ra-atomics.chpl        : a version of RA using buffered atomic vars
```

Comment:
Did you mean "ra-buff-atomics" ?

Reply:
Whoops, also a good catch. Force pushed the correction